src/Tizen.NUI/src/public/Common/StyleManager.cs

```diff
@@ -106,6 +117,34 @@ public static StyleManager Get()
             return ret;
         }
 
+        /// <summary>
+        /// Sets the brokenImageUrl.
+        /// When the broken image and type are set in the Application,
```

Comment:
What is `broken image`? is it a commonly used term?

Reply:
BrokenImage is the image to show when image loading is failed
currently, we just show the broken Image that dali has internally.
But Application wants to set their broken image directly.
so I added them.

Reply:
You should mention about broken image on the summary

Reply:
Okay, i will add it